windows/src/desktop/kmshell/main/Keyman.System.DownloadUpdate.pas

```diff
@@ -0,0 +1,203 @@
+(*
+ * Keyman is copyright (C) SIL Global. MIT License.
+ *)
+unit Keyman.System.DownloadUpdate;
+
+interface
+uses
+  System.Classes,
+  System.SysUtils,
+  httpuploader,
+  Keyman.System.UpdateCheckResponse,
+  KeymanPaths,
+  OnlineUpdateCheck;
+
+const
+  DaysBetweenCheckingForUpdates: Integer = 7; // Days between checking for updates
```

Comment:
```suggestion
```

According to https://github.com/keymanapp/keyman/pull/12079/files#r1824343506, this is no longer needed?